pkgs/tools/video/yamdi/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  stdenv,
+  fetchurl,
+}:
+
+stdenv.mkDerivation rec {
+  name = "yamdi-${version}";
+  version = "1.9";
+
+  # Source repo is also available here:
+  # https://github.com/ioppermann/yamdi
+  src = fetchurl {
+    url = "https://downloads.sourceforge.net/project/yamdi/yamdi/${version}/yamdi-${version}.tar.gz";
```

Comment:
Ah, ok.

Re: fetchurl/fetchgit/fetchFromGitHub: I was unsure which among those are preferred when all are available. I also couldn't tell in this case if the github repo was intended to be always up to date, or just a mirror of the sourceforge one, but the project homepage was hosted on sourceforge, so I went with that. Is there a general approach to take in a case like that?